drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -78,6 +84,14 @@ func (e *errLabelAbsent) Error() string {
 
 type k8sSchedOps struct{}
 
+func (k *k8sSchedOps) EnableOnNode(n node.Node) error {
```

Comment:
instead of enable and disable, let's name them appropriately as start and stop as that's what the APIs do.

In the past, they might have been named incorrectly.

Reply:
Fixed. 
`EnableOnNode ==> StartPxOnNode`
`DisableOnNode  ==> StopPxOnNode`